tests/test_clos/test_get_job_status.py

```diff
@@ -0,0 +1,71 @@
+"""Pytest added for function get_job_status"""
+import json
+import mock
+import pytest
+import requests
+import responses
+from cloudos.clos import Cloudos
+from cloudos.utils.errors import BadRequestException
+from tests.functions_for_pytest import load_json_file
+
+INPUT = "tests/test_data/process_job_list_initial_json.json"
```

Comment:
As @danielboloc suggested, it would be better to work with a more realistic response from the server, rather than using the response from another GET command. In addition. This way we will have it here for future reference.

Reply:
trying to do this